pipelines/rj_segur__forca_municipal/task.py

```
@@ -1,173 +1,81 @@
-from pipelines.rj_segur__forca_municipal import env
 
-SP_TZ = ZoneInfo("America/Sao_Paulo")
-PAGE_SIZE = 500
-
-
-class FMApi:
-    """Cliente HTTP para a API CIVITAS/HxGN OnCall (Força Municipal)."""
+def _content_hash(df: pd.DataFrame) -> str:
+    """Gera um hash MD5 truncado (12 chars) do conteúdo bruto do DataFrame."""
+    raw = pd.util.hash_pandas_object(df, index=False).values.tobytes()  # type: ignore[attr-defined]
+    return hashlib.md5(raw).hexdigest()[:12]
```

Ruff (0.15.12)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)